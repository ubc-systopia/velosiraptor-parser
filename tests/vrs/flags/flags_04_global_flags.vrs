flags { 1,  bar }
