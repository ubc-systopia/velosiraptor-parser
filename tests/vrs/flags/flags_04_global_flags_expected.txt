error: expected `}`, found `1`
     --> tests/vrs/flags/flags_04_global_flags.vrs:1:9
      |
    1 |  flags { 1,  bar }
      |          ^ help: add `}` here.
